pkg/connectivity/pod-to-pod.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/openservicemesh/osm-health/pkg/kubernetes/namespace"
 	"github.com/openservicemesh/osm-health/pkg/kubernetes/pod"
 	"github.com/openservicemesh/osm-health/pkg/kubernetes/podhelper"
-	"github.com/openservicemesh/osm-health/pkg/osm"
+	"github.com/openservicemesh/osm-health/pkg/osm/utils"
```

Comment:
I'm typically not a huge fan of using packages like `utils` - it becomes the `misc` package.
Not a big deal, but there's got to be a better place for this. Seems like `osm` was an ok fit too.